subsys/nrf_security/src/ssf_secdom/ssf_crypto.c

```diff
@@ -0,0 +1,476 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+#include <psa/crypto.h>
+#include <sdfw/sdfw_services/crypto_service.h>
+
+psa_status_t psa_crypto_init(void)
+{
+	return ssf_psa_crypto_init();
+}
+
+psa_status_t psa_get_key_attributes(mbedtls_svc_key_id_t key, psa_key_attributes_t *attributes)
+{
+	return ssf_psa_get_key_attributes(key, attributes);
```

Comment:
I can't find this function.

I assume this PR is not complete yet, i.e. I can't checkout this PR and build ssf_crypto.c at the moment?

Reply:
Right. This file is not updated with the latest PSA functions. It depends on another PR. I will update this once the other PR is merged.